### ISSUE TITLE
Pixelated subtitles in videos with low resolution.

### ISSUE THREAD

**Issue by vdualb** [Rafostar/clapper#475]
Try to add this subtitles file [subtitles.txt](https://github.com/user-attachments/files/16319312/subtitles.txt) (change extension to vtt, i couldn't upload .vtt file to github) to this video [video.webm](https://github.com/user-attachments/assets/46d5ef94-9c8c-436c-84bf-bc8890c3d112)
Clapper:
![image](https://github.com/user-attachments/assets/15a8b67f-ba16-47c0-b9ee-77091c19e340)
VLC:
![image](https://github.com/user-attachments/assets/2b1a68e5-161c-4487-bb4d-0127ca84f7f1)

**Comment by peteruithoven** [comment]
This seems like a duplicate of https://github.com/Rafostar/clapper/issues/321 ?

**Comment by Rafostar** [comment]
Yeah, but adding subtitles to a video that has 36x36 pixels resolution is a very "unique" case. Will be useful to remember to test this, so I am not closing this issue.
